node/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "private": "true",
+  "dependencies": {
+    "@googlemaps/google-maps-services-js": "^3.1.6"
+  },
+  "devDependencies": {
+    "@types/googlemaps": "^3.39.13",
+    "@vtex/api": "6.36.3",
+    "vtex.geolocation-graphql-interface": "https://w0geo--checkoutio.myvtex.com/_v/private/typings/linked/v1/vtex.geolocation-graphql-interface@0.0.2+build1602857548/public/@types/vtex.geolocation-graphql-interface",
```

Comment:
Why this? See https://github.com/vtex/builder-hub/pull/1065

Although this is weird, should we...

1. Keep it that way (I don't think it will break)
2. The next PR fixes it (probably  `geolocation-graphql-interface` will be fixed already)

OR

1. Wait DEFCON 3 ends and deploy the `builder-hub` fix
2. Release a new dummy version for `geolocation-graphql-interface` (so the `builder-hub` fix will be applied)
3. Updates this line and merge changes